extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -214,9 +221,16 @@ describe('amp-a4a', () => {
     expect(nameData).to.be.ok;
     verifyNameData(nameData);
     expect(child).to.be.visible;
+    const sandboxAttribute = child.getAttribute('sandbox');
+    if (shouldSandbox) {
+      expect(sandboxAttribute).to.be.ok;
```

Comment:
Consider verifying the value of the attribute?

Reply:
Done.